templates/legal/data-privacy/sso.md

```diff
@@ -15,7 +15,7 @@ This privacy notice tells you about the information collected from you when you
 
 ## Who are we?
 
-We are Canonical Group Limited. Our address is 2nd Floor, Clarendon House, Victoria Street, Douglas IM1 2LN, Isle of Man. You can contact us by post at the above address or by email to [dataprotection@canonical.com](mailto:dataprotection@canonical.com) for the attention of "Legal".
+We are Canonical Group Limited. Our address is 5 New Street Square, London, EC4A 3TW, United Kingdom. You can contact us by post at the address set out in the “right to complain” section, below or by email to [dataprotection@canonical.com](mailto:dataprotection@canonical.com) for the attention of "Legal".
```

Comment:
The version date isn't updated.

The address on the copy doc is the one on "201 Borough High St." and the "set out in the “right to complain” section, below" part isn't added.

Also, it seems there were some updates that weren't implemented, references to Ubuntu One. They should probably be handled in another task.

Reply:
The wrong address here. The doc says "The Office Group, St Dunstans House, 4th floor, 201 Borough High St, London SE1 1JA", unless that's a mistake